curriculum/challenges/english/25-front-end-development/lecture-working-with-css-libraries-and-frameworks/67d1d74780981f521b8ac090.md

```diff
@@ -2,14 +2,84 @@
 id: 67d1d74780981f521b8ac090
 title: What Is a CSS Framework, and What Are Some Advantages and Disadvantages of Using It?
 challengeType: 11
-videoId: nVAaxZ34khk
+videoId: A-z2RMOP9dE
 dashedName: what-is-a-css-framework-and-what-are-some-advantages-and-disadvantages-of-using-it
 ---
 
 # --description--
 
 Watch the video lecture and answer the questions below.
```

Comment:
this needs to be updated to the new message 

https://github.com/freeCodeCamp/freeCodeCamp/issues/57750#issuecomment-2762774615